### DISCUSSION TITLE
Is it possible to enter the bioavailability of oral drugs in PK-sim?

### DISCUSSION THREAD

**Discussion by hong77777** [Open-Systems-Pharmacology/Forum#1023] [Questions & Problems] [answered]
Hi guys, I am working on a drug where the bioavailability of the inactive prodrug is only 7% for oral administration and it is the active form that really works. I would like to know if bioavailability can be addressed in this software.

**Comment by prvmalik** [comment]
Yes, you can add a metabolic enzyme into the gut lumen, the gut mucosa, or the liver to describe conversion to the active form and optimize to 7% prodrug using an observer that tracks the amount of prodrug in the body as a percentage of dose administered

**Comment by hong77777** [comment]
> 
> 
> 是的，您可以将代谢酶添加到肠腔、肠粘膜或肝脏中，以描述转化为活性形式的过程，并使用观察者跟踪体内前药量占剂量百分比的情况优化为 7% 前药管理

Sorry，I don't understand the meaning of "optimize to 7% prodrug using an observer that tracks the amount of prodrug in the body as a percentage of dose administered".

**Comment by StephanSchaller** [comment, marked as answer]
Dear @hong77777, how/where is the prodrug converted/metabolized to its active form?

Biovailability can be controlled/defined with/by multiple properties. limited solubility, limited intestinal permeability (or active efflux), intestinal metabolism (to name the major ones). This can all be defined for your compound in the software, but we need a little more info about the (pro)drug to help you.

Best,
Stephan

**Reply by hong77777** [reply]
Thank you very much for your help!
